GETTING_STARTED.md

```diff
@@ -461,19 +461,19 @@ generate :email
 # => "person2@example.com"
 ```
 
-Sequences can be used as attributes:
+Sequences can be used in lazy attributes:
 
 ```ruby
-factory :user do
-  email
+factory :invite do
+  invitee { generate(:email) }
 end
 ```
 
-Or in lazy attributes:
+Or as attributes:
```

Comment:
@jaredbeck we refer to these as implicit attributes; mind including that clarification?